hazelcast/src/main/java/com/hazelcast/util/scheduler/EntryTaskScheduler.java

```diff
@@ -47,5 +48,14 @@
 
     void cancelAll();
 
+    /**
```

Comment:
I do not think that yours is more useful than adding an extra method. As I noted in the PR message, that method is mainly used for debugging and testing. What it does it providing a snapshot of its internal state. It is not introducing a new class / interface or anything else. You can get the same information by calling other methods for each partition. So it is not introducing anything new. Additionally, it is not called from anywhere else expect test util methods. I think this is OK. If my understanding is correct, what you do is adding extra logic, which is not necessary other than testing purposes, to already existing methods.